libs/server-sdk/src/data_components/expiration_tracker/expiration_tracker.cpp

```diff
@@ -87,8 +87,9 @@ ExpirationTracker::TrackState ExpirationTracker::State(TimePoint expiration,
 void ExpirationTracker::ScopedTtls::Set(DataKind kind,
                                         std::string const& key,
                                         TimePoint expiration) {
-    data_[static_cast<std::underlying_type_t<DataKind>>(kind)].Data().insert(
-        {key, expiration});
+    data_[static_cast<std::underlying_type_t<DataKind>>(kind)]
+        .Data()
+        .insert_or_assign(key, expiration);
```

Comment:
The previous behavior was that if the key already existed, this was a no-op. I think that's incorrect since that means you can't extend (update) the expiry of an item. But perhaps I am holding it wrong.